files/image_config/procdockerstats/procdockerstats

```diff
@@ -0,0 +1,212 @@
+# !/usr/bin/env python
+'''
+procdockerstats
+Daemon which periodically gathers process and docker statistics and pushes the data to STATE_DB
+'''
+
+import sys
+import os
+import time
+import syslog
+import subprocess
+import re
+import swsssdk 
+from datetime import datetime
```

Comment:
I'm planning to standardize the order with which we import files in SONiC Python files. Can you please update this section to match the following:

```
import os
import re
import subprocess
import sys
import syslog
import time
from datetime import datetime

import swsssdk 
```